docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -135,7 +135,12 @@ property key | Description | Default |
 `correlation.element_max_number`|Max element count of the correlation context.|`3`|
 `correlation.value_max_length`|Max value length of correlation context element.|`128`|
 `plugin.kafka.bootstrap_servers`| A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | `localhost:9092`
-`plugin.kafka.consumer_config`| Kafka producer configuration. |
+`plugin.kafka.consumer_config`| Kafka producer configuration. ||
+`plugin.dubbo.collect_consumer_arguments`| Apache Dubbo consumer collect `arguments` in RPC call. |`false`| 
+`plugin.dubbo.consumer_arguments_length_threshold`| When `plugin.dubbo.collect_consumer_arguments` is `true`, Arguments of length from the front will to the OAP backend |`256`| 
+`plugin.dubbo.collect_provider_address`| Apache Dubbo provider collect `remote_address` and `local_address` in RPC call. |`false`| 
```

Comment:
Notice, the provider side is EntrySpan, which has `#setPeer` too. Please recheck.